templates/plugin/dynflow_core.yml.erb

```diff
@@ -0,0 +1,61 @@
+---
+# Path to dynflow database, leave blank for in-memory non-persistent database
+:database: <%= scope.lookupvar('::foreman_proxy::plugin::dynflow::database_path') %>
+:console_auth: <%= scope.lookupvar('::foreman_proxy::plugin::dynflow::console_auth') %>
+
+# URL of the foreman, used for reporting back
+:foreman_url: <%= scope.lookupvar("foreman_proxy::foreman_base_url") %>
+
+# SSL settings for client authentication against foreman.
+<% if scope.lookupvar("foreman_proxy::foreman_ssl_ca") -%>
```

Comment:
Tests are failing one some platforms due to this check, we probably need to go this approach to cover all the posible nil/undef values https://github.com/theforeman/puppet-foreman_proxy/blob/5f0957b1d7f3d2818903c6c824209a773cc738ae/templates/settings.yml.erb#L72